index.js

```diff
@@ -458,6 +466,18 @@ module.exports = {
     'operator-linebreak': 'warn',
     // enforce padding within blocks
     'padded-blocks': ['warn', 'never'],
+    // require or disallow padding lines between statements
+    'padding-line-between-statements': ['warn',
+      { blankLine: 'always', prev: '*', next: 'return' },
```

Comment:
This line seems to replace the `newline-before-return` rule, which we had set to `off` previously, so I think we want to remove this configuration here, unless it's a new rule we want to start enforcing.